style.css

```diff
@@ -78,8 +83,182 @@ nav .menu-button {
   margin-left: 24px;
 }
 
+/* style social medial icons */
 .social-icons {
   display: flex;
   gap: 20px;
-  margin-top: 10px;
+  margin: 10px 0 0 10px;
+}
+
+/* position work section */
+.work-section {
+  position: relative;
+  top: 90%;
+}
+
+/* style work section part */
+.projects {
+  margin: 114px auto;
+  background-color: white;
+  height: 585px;
+  width: 327px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+/* positon and size projects snapshoot */
+.project-image {
+  height: 220px;
+  margin: 12px 12px;
+}
+
+/* position projects description */
+.projects-describe {
+  margin-left: 12px;
+}
+
+/* style project description */
+.project-title {
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+  font-weight: 700px;
+  width: 295px;
+}
+
+/* postion project frame */
+.projects-describe .frame {
+  display: flex;
+  gap: 32px;
+  margin-top: 16px;
+}
+
+/* style projects description */
+.description {
+  margin-top: 16px;
+  line-height: 24px;
+  font-weight: 400px;
+  font-size: 15px;
+  width: 295px;
+  height: 76px;
+  color: #344563;
+}
+
+/* postion technologies used */
+.tools {
+  display: flex;
+  gap: 12px;
+  margin-top: 24px;
+}
+
+/* style technologies used */
+.tools li {
+  height: 24px;
+  font-size: 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  text-align: center;
+  padding: 4px 12px;
+  font-weight: 500px;
+  color: #6070ff;
+}
+
+/* style all the clickable buttons */
+.projects .button,
+.resume-button {
+  padding: 12px 20px;
+  width: 100px;
+  height: 24px;
+  font-weight: 500px;
+  size: 17px;
+  line-height: 48px;
+  color: #396df2;
+  text-align: center;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  position: relative;
+  top: 25px;
+}
+
+/* resize project 3 and 4 size */
+.project2,
+.project4 {
+  height: 630px;
+}
+
+/* postion about me section */
+.about {
+  width: 375px;
+  height: 1100px;
+  background: white;
+  border: 1px solid white;
+  border-radius: 0 80px 0 0;
+}
```

Comment:
- [ ] Kindly use flexbox to align the contents of the about section as demanded by the project requirement. This is best for the responsiveness of your work.